spring-cloud-dataflow-module-deployer-spi/src/main/java/org/springframework/cloud/dataflow/module/ModuleStatus.java

```diff
@@ -79,7 +79,7 @@ public DeploymentState getState() {
 		}
 
 		if (states.size() == 0) {
-			return DeploymentState.undeployed;
+			return DeploymentState.unknown;
```

Comment:
I had a discussion with @markfisher and @pperalta on Slack (https://pivotal.slack.com/archives/spring-cloud-stream/p1452008030001835) about this. I believe this is ok, given https://github.com/spring-cloud/spring-cloud-dataflow/blob/36e97d667d9ec04be4e9dcb6f8a28ac60a0eab0d/spring-cloud-dataflow-admin-starter/src/main/java/org/springframework/cloud/dataflow/admin/controller/StreamDefinitionController.java#L206-L208